crates/bevy_ecs/src/schedule_v3/set.rs

```diff
@@ -23,6 +23,12 @@ pub trait SystemSet: DynHash + Debug + Send + Sync + 'static {
         false
     }
 
+    /// Returns `true` if this set is a "base system set", which systems
```

Comment:
Phrasing is particularly awkward. See general comments about a central explanation.